set.mm

```diff
@@ -139051,6 +139053,12 @@ Infinity and the extended real number system (cont.)
     cle xrleloe mpan mpbid ord eqcom syl6ib impbid2 ) ABCZADEZDAFGZHZUFUGDDFGZD
     BCZUIHIDJKADDFLMUEUHDAEZUFUEUGUKUEDAQGZUGUKNZAOUJUEULUMPIDARSTUADAUBUCUD $.
 
+  $( An extended real which is less than minus infinity is minus infinity.
```

Comment:
In English, this should be "less than or equal to minus infinity", or "at most minus infinity". (Your phrasing would be correct in French, where "less than" means `<_` !)